src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleaseHelpAndSupportSection.tsx

```diff
@@ -36,118 +31,54 @@ const PublicationReleaseHelpAndSupportSection = ({
-          <AccordionSection heading="Official statistics" headingTag="h3">
-            <OfficialStatisticsSection />
-          </AccordionSection>
-        )}
-        {releaseType === 'AdHocStatistics' && (
-          <AccordionSection
-            heading="Ad hoc official statistics"
-            headingTag="h3"
-          >
-            <AdHocOfficialStatisticsSection />
-          </AccordionSection>
-        )}
-        {releaseType === 'ExperimentalStatistics' && (
-          <AccordionSection heading="Experimental statistics" headingTag="h3">
-            <ExperimentalStatisticsSection />
-          </AccordionSection>
-        )}
-        {releaseType === 'ManagementInformation' && (
-          <AccordionSection heading="Management information" headingTag="h3">
-            <ManagementInformationSection />
-          </AccordionSection>
-        )}
-        <AccordionSection
-          heading="Contact us"
-          headingTag="h3"
-          caption="Ask questions and provide feedback"
-          id="contact-us"
-        >
-          <ContactUsSection
-            publicationContact={publicationContact}
-            publicationTitle={publicationTitle}
-          />
-        </AccordionSection>
-      </AccordionComponent>
-    </>
-  );
-};
 
-interface AccordionComponentProps {
-  accordionId: string;
-  includeAnalytics: boolean;
-  publicationTitle: string;
-  children: ReactNode;
-}
+      {(methodologies.length || externalMethodology) && (
+        <>
+          <h3>Methodology</h3>
+          <p>
+            Find out how and why we collect, process and publish these
+            statistics.
+          </p>
 
-const AccordionComponent = ({
-  accordionId,
-  includeAnalytics,
-  publicationTitle,
-  children,
-}: AccordionComponentProps) => {
-  return includeAnalytics ? (
-    <Accordion
-      toggleAllHiddenText="help and support sections"
-      id={accordionId}
-      onSectionOpen={accordionSection => {
-        logEvent({
-          category: `${publicationTitle} help and support`,
-          action: 'Accordion opened',
-          label: accordionSection.title,
-        });
-      }}
-    >
-      {children}
-    </Accordion>
-  ) : (
-    <Accordion id={accordionId} toggleAllHiddenText="help and support sections">
-      {children}
-    </Accordion>
+          <ul className="govuk-list govuk-list--spaced">
+            {methodologies.map(methodology => (
+              <li key={methodology.id}>
+                <Link to={`/methodology/${methodology.slug}`}>
+                  {methodology.title}
+                </Link>
+              </li>
+            ))}
+            {externalMethodology && (
+              <li>
+                <Link to={externalMethodology.url}>
+                  {externalMethodology.title}
+                </Link>
+              </li>
+            )}
+          </ul>
+        </>
+      )}
+      {releaseType === 'NationalStatistics' && <NationalStatisticsSection />}
```

Comment:
Part of me wonders if we could create a helper function to simplify the logic of checking what the release type is and what section we should render but think its out of scope for this PR

Reply:
Yeah we probably could, but I'd prefer to keep this PR just to govuk-frontend related changes.